### DISCUSSION TITLE
Are there orders in hdam_id and stay_id in mimic IV?

### DISCUSSION THREAD

**Discussion by hhchang0210** [MIT-LCP/mimic-code#1457] [MIMIC-IV]
### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Description of the issue, including:
* what you have tried
* references to similar issues
* queries demonstrating your question (if applicable)

I know a subject_id may have many hadm_id and stay_id. But I am curious if there are orders in hadm_id and stay_id.
What I want to do is to parse diagnoses from diagnoses_icd stable and I want to know what diagnoses the patient has before this admission.  For example, a subject_id: 10000032 has hadm_ids: 22595853 and 22841357. Is the time of 22841357 newer than the time of 22595853?

**Comment by alistairewj** [comment]
You can order `hadm_id` and `stay_id` but not by the integer value. Instead, use a time from the table which provides information about the column. So for `hadm_id`, you can sort them by `admittime` in the *admissions* table. Similarly, for `stay_id`, you can sort them by `intime` in the *icustays* table.
